content/smart.md

```diff
@@ -0,0 +1,140 @@
+---
+title: SMART
+layout: smart
+---
+
+# SMART<sup>®</sup> #
+------------------------------------------------------------------------
+* TOC
+{:toc}
+
+## General ##
+
+### Introduction ###
+The SMART<sup>®</sup> (Substitutable Medical Apps and Reusable Technology) platform defines a specification for an electronic health record (EHR) to safely and securely open other applications with context. These SMART applications are commonly web applications but may also be native mobile applications and that use HL7® FHIR® [standard][HL7-FHIR] to read and write data from the EHR. With SMART, Cerner can embed a SMART app in the EHR. Cerner believes that SMART applications will be a major user of FHIR resources. We will also support FHIR access through mobile SMART applications, as those specifications emerge from the SMART [web site][SMART-PLATFORM].
+
+### SaaS Model ###
+
+Cerner expects the majority of SMART apps to be provided to clients in a SaaS (software as a service) model. That is, the SMART app is hosted or managed by the SMART developer or provider. In this case, clients do not need to install any code or package in order to implement a particular SMART app.
+
+### Authorization Model ###
+
+At a high level, provider facing applications must receive the following authorizations:
+
+1. Cerner must validate and register every SMART app. As part of this process, Cerner decides which FHIR resources and operations (for example, READ and WRITE) the SMART app is allowed to use.
+2. Each client decides whether they want to allow a SMART app to run on a domain-by-domain basis.
+3. Each client decides which users and roles can access the SMART app. Today, this access is determined with a configuration.
```

Comment:
For bullet 1 Cerner doesn't decide the resources developers use. We validate the usage of the resources. We also won't be validating resouce usage for direct to consumer apps.